src/main/scala/cromwell/binding/Scope.scala

```diff
@@ -1,14 +1,168 @@
 package cromwell.binding
 
+import cromwell.binding.AstTools.AstNodeName
+import cromwell.parser.WdlParser.{AstList, Ast, AstNode}
+import scala.collection.JavaConverters._
+
+
+object Scope {
+
+  /**
+   * Utility class to generate a ScopeTree from an Ast.
```

Comment:
My spidey sense starts tingling when I see phrases like "utility class" in scala